VideoWeb/VideoWeb/ClientApp/src/sass/abstracts/_mixins.scss

```diff
@@ -0,0 +1,27 @@
+@import 'functions';
+@import 'breakpoints';
+
+
+@mixin responds-to($name) {
+    @if not map-has-key($breakpoints, $name) {
+      @warn "Warning: `#{$name}` is not a valid breakpoint name.";
+    } @else {
+      @media only screen and (min-width: em(map-get($breakpoints, $name))) {
+        @content;
+      }
+    }
+  }
+
+  @mixin responds-to-only($name) {
+    @if not map-has-key($breakpoints, $name) {
+      @warn "Warning: `#{$name}` is not a valid breakpoint name.";
+    } @else {
+      $next: map-get-next($breakpoints, $name);
+      @if not $next {
+        @warn "Warning: `#{$name}` is the last breakpoint.";
+      }
+      @media only screen and (min-width: em(map-get($breakpoints, $name))) and (max-width: em($next - 1px)) {
+        @content;
+      }
+    }
+  }
```

Comment:
I'm going to check and see if this can be made more DRY as it repeats the mixin above a lot.